### PR TITLE
Advent of code

### LINKED ISSUE #9334: advent of code
### Brand Name

Advent of Code

### Website

https://adventofcode.com/

### Popularity Metric

> 159,497
https://www.similarweb.com/website/adventofcode.com/#overview

### Official Resources for Icon and Color

So there is no official icon for advent of code I assumed we could use their favicon
[
favicon
](url)
the official colors for this favicon:
#0a0e25
and 
#ffff55
I believe it would be more fitting to use 

### Additional Comments

_No response_

### Contributing

No

### PR DESCRIPTION
**Pull request by 21st-centuryman**
**Issue:** closes #9334

**Similarweb rank:**
> 205,048 
https://www.similarweb.com/website/adventofcode.com/#overview

This number is definitely going to go up this Christmas 🎄 

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
adventofcode

The entire website was based on the 'Source Code Pro slim' font, the logo for the favicon looked identical so i just used that for the icon. Also I was skeptical if I should use the signature blue or yellow, but the blue looked much better.

If you have any notes I would love to hear it

### CODE REVIEW

**Comment by runxel** [comment]
Thanks for your time to contribute.
I'm torn on the color, tho. The star itself is this bright yellow, so I personally think this would only work, if we make the logo negative, with the star being cut out.
The star however should be yellow. Any from the @simple-icons/maintainers might chime in please for decision?

**Comment by 21st-centuryman** [comment]
So I was not sure that negatives were allowed. I decided to take the time and just create 3 more previews. Let me know what you guy's prefer:
adventofcode(2)
adventofcode(4)

Blue negative was unwatchable in the preview so I will not show that.

**Comment by 21st-centuryman** [comment]
ow when i see it I think the yellow logo is better, I pushed it

File: _data/simple-icons.json [hunk 1]
```diff
@@ -278,6 +278,11 @@
             "hex": "0DBDFF",
             "source": "https://www.adroll.com"
         },
+        {
+            "title": "Advent Of Code",
+            "hex": "FFFF66",
```
Comment (runxel) [inline _data/simple-icons.json:283]: I'm getting #ffff55 as a color tho... How did you extracted it?
I used the Firefox dev tool integrated color meter. Getting colors from rasters are always a slight gamble, so I would let it pass.

Comment (21st-centuryman) [reply _data/simple-icons.json:283]: I checked the font color of the completed stars in my events. 
<span class="calendar-mark-complete">. I had to double check earlier just to make sure

Comment (runxel) [reply _data/simple-icons.json:283]: Thank you so much for this quick explanation. I had looked into the favicon, where I got that color in the middle of the star. But having the hex seems more deliberate to me.

**Pull request merged**